docs/user_guide.md

```diff
@@ -198,6 +202,16 @@ If the `max_tokens` value in your `llm_config` exceeds the actual limit of the m
 
 See [Azure OpenAI Quickstart](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart?tabs=keyless%2Ctypescript-keyless%2Cpython-new%2Ccommand-line&pivots=programming-language-python) for more information.
 
+### Anthropic
+
+To use Anthropic models, set the `ANTHROPIC_API_KEY` environment variable to your Anthropic API key
+and specify which model to use in the `model_name` field of the `llm_config` section of an agent network hocon file:
+```hocon
+    "llm_config": {
+        "model_name": "claude-3-5-haiku",
+    },
+```
+You can get an Anthropic API key [here](https://console.anthropic.com/settings/keys)
```

Comment:
You might want to point at the default_llm_info.hocon so people know what other anthropic models are available.